documentation/general/contributing.md

```diff
@@ -0,0 +1,178 @@
+# Welcome
+
+Thank you for your interest in Openverse! We're so excited to bring new
+contributors into the fold in Openverse, WordPress and FOSS in general. This
+document is a set of guidelines to help you contribute to this project.
+
+## Code of Conduct
+
+You should read and agree to abide by the [code of conduct](CODE_OF_CONDUCT.md)
+before contributing to WordPress projects. This applies to all Openverse
+repositories because Openverse is a WordPress Foundation project.
+
+## Keep in touch
+
+Don't hesitate to ask for help; if you're stuck, we're here for you! You can
+ping us via any of our [communication channels](README.md#keep-in-touch).
+
+## Code and code-related contributions
+
+### Get started
+
+Detailed help for contributing code can be found in the
+[developer documentation](https://docs.openverse.org/), which also includes the
+following subfolders.
+
+- [Storybook](https://docs.openverse.org/storybook/) (UI components)
+- [Tailwind Config Viewer](https://docs.openverse.org/tailwind/) (design tokens)
```

Comment:
This is covered in the root of the documentation (https://github.com/WordPress/openverse/blob/main/documentation/index.md#related-documentation), so this section can be deleted.